src/components/shared/SearchCard/SearchCardHeader.js

```diff
@@ -35,7 +32,12 @@ export const SearchCardHeader = ({ experiment, isLinked = false }) => {
       >
         {isLinked ? (
           <Anchor
-            href={`experiments/${accessionCode}/${formatURLString(title)}`}
+            href={{
+              pathname: `experiments/${accessionCode}/${formatURLString(
+                title
+              )}`,
+              query: { ref: 'search' }
+            }}
```

Comment:
Removed the use of `useSearchManager` for the search page reference and adjusted `gtag.trackLink` accordingly.